source/lib/predicates/number.ts

```diff
@@ -141,4 +141,82 @@ export class NumberPredicate extends Predicate<number> {
 			validator: value => is.integer(value) || is.infinite(value)
 		});
 	}
+
+	/**
+	 * Test a number to be in a valid range for a uint8.
```

Comment:
I think we should use the full naming here:

> Test a number to be in a valid range for a 8-bit unsigned integer.

Reply:
Done